tests/validator/validator.cpp

```diff
@@ -195,6 +189,14 @@ TEST(Validator, unnamedAndDuplicateNamedVariablesWithAndWithoutValidUnits)
     v4->setUnits("dollars");
     v5->setName("mullah");
     v5->setUnits("$$");
+
```

Comment:
Again I don't understand the need for the change?  Is there a difference in behaviour? (and if so, eek?) Also in other tests.

Reply:
028240c